pulseapi/entries/views.py

```diff
@@ -64,7 +64,7 @@ def toggle_featured(request, entryid, **kwargs):
     """
     user = request.user
 
-    if user.has_perm('entries.can_change_entry'):
+    if user.has_perm('entries.change_entry'):
```

Comment:
`can_change_entry` was programmatically created in the tests, but the actual permission for changing entries has the codename `change_entry`